README.md

```diff
@@ -98,10 +95,8 @@ has instructions for running the samples.
 
 | Sample                      | Source Code                       | Try it |
 | --------------------------- | --------------------------------- | ------ |
-| Add Task | [source code](https://github.com/googleapis/nodejs-datastore/blob/master/samples/addTask.js) | [![Open in Cloud Shell][shell_img]](https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/googleapis/nodejs-datastore&page=editor&open_in_editor=samples/addTask.js,samples/README.md) |
```

Comment:
@bcoe I am not sure what's happening here.  I added a bunch of samples with names like `tasks.add.js`, each with their own sample metadata ... but none of them are showing up here.

Reply:
https://github.com/googleapis/synthtool/pull/261